<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Agregar Venta</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Cliente :</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QPushButton" name="btn_AgregarVenta">
           <property name="text">
            <string>Agregar Venta</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="comboBox_Clientes">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Producto :</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="input_Cantidad"/>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="comboBox_Productos"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Cantidad :</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_EliminarVentaDeTabla">
           <property name="text">
            <string>Eliminar Venta de Tabla</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_GenerarPDF_TablaVentas">
           <property name="text">
            <string>Exportar Tabla</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_RegistroStock">
           <property name="text">
            <string>Registro Stock</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_RegistroClientes">
           <property name="text">
            <string>Registro Clientes</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Salir">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Salir</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableWidget" name="table_Ventas">
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Hora de Registro</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Cliente</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Producto</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Cantidad</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Precio Unidad</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Precio Total</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>TOTAL :</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_Total">
          <property name="text">
           <string>XXXX</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Fecha de hoy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_fechaDeHoy">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>DD/MM/AA</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Precio Productos</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_28">
             <item>
              <widget class="QLabel" name="label_31">
               <property name="text">
                <string>Bolsa Hielo Rolito  4kg --&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Precio_BolsaHieloRolito4KG">
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_25">
             <item>
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>Bolsa Hielo Rolito  10kg --&gt; </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Precio_BolsaHieloRolito10KG">
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>Bolsa Hielo Rolito  14kg --&gt; </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Precio_BolsaHieloRolito14KG">
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_24">
             <item>
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>Bolsa Hielo Molido  14kg --&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Precio_BolsaHieloMolido14KG">
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_26">
             <item>
              <widget class="QLabel" name="label_29">
               <property name="text">
                <string>Agua destilada 1L --&gt; </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Precio_AguaDestilada1L">
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21">
             <item>
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>Agua destilada 5L --&gt; </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Precio_AguaDestilada5L">
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_22">
             <item>
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>Agua destilada Pack 6 de 5L --&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Precio_Pack6AguaDestilada">
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>Agua destilada Pack 8 de 1L --&gt; </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Precio_Pack8AguaDestilada">
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20">
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Liquido Refrigerante 1L --&gt; </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Precio_LiquidoRefrigerante1L">
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_27">
             <item>
              <widget class="QLabel" name="label_30">
               <property name="text">
                <string>Liquido Refrigerante 5L --&gt; </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Precio_LiquidoRefrigerante5L">
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Liquido Refrigerante Pack 6 de 5L --&gt; </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Precio_Pack6LiquidoRefrigerante">
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_30">
             <item>
              <widget class="QLabel" name="label_33">
               <property name="text">
                <string>Agua destilada Granel 1L --&gt; </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Precio_AguaDestiladaGranel1L">
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_29">
             <item>
              <widget class="QLabel" name="label_32">
               <property name="text">
                <string>Liquido Refrigerante Granel 1L --&gt; </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Precio_LiquidoRefrigeranteGranel1L">
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>comboBox_Clientes</tabstop>
  <tabstop>comboBox_Productos</tabstop>
  <tabstop>input_Cantidad</tabstop>
  <tabstop>btn_AgregarVenta</tabstop>
  <tabstop>btn_EliminarVentaDeTabla</tabstop>
  <tabstop>btn_GenerarPDF_TablaVentas</tabstop>
  <tabstop>btn_RegistroStock</tabstop>
  <tabstop>btn_RegistroClientes</tabstop>
  <tabstop>btn_Salir</tabstop>
  <tabstop>table_Ventas</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
